controllers/cards.js

```diff
@@ -0,0 +1,56 @@
+const Card = require('../models/card');
+const NotFoundError = require('../errors/notFoundError');
+
+const getAllCards = ((req, res) => {
+  Card.find({})
+    .then((card) => res.send({ data: card }))
+    .catch(() => res.status(500).send({ message: 'Произошла ошибка' }));
+});
+
+const createCard = ((req, res) => {
+  const { name, link } = req.body;
+  Card.create({ name, link, owner: req.user._id })
+    .then((newCard) => res.send({ data: newCard }))
+    .catch((err) => res.status(500).send({ message: err.message }));
+});
+
+const deleteCard = async (req, res) => {
+  await Card.findByIdAndRemove(req.params.cardId)
```

Comment:
Нужно исправить: при удалении несуществующей карточки возвращается 200.
Это связано с особенностью метода `.findByIdAndRemove()`. Команда преобразуется в [`.findAndModify()`](https://docs.mongodb.com/manual/reference/method/db.collection.findAndModify/) и с ней колбэк `.orFail()` не работает.

```suggestion
  await Card.findByIdAndDelete(req.params.cardId)
```